charts/caraml-store/templates/serving/secret.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: {{ template "caraml-store.serving.fullname" . }}
   namespace: {{ .Release.Namespace }}
   labels:
-    component: registry
+    component: serving
```

Comment:
CMIIW we can move this label to the template, right?

Reply:
Yes, we can do so. Though i am also contemplating to just drop this `component` label altogether. Can't think of much use case where the user need to add labels to secrets/configmap.